resources/dicts/conditions/pregnancy.json

```
@@ -38,7 +52,11 @@
     " {PRONOUN/m_c/subject/CAP} {VERB/m_c/aren't/isn't} keen on being confined to the nursery just yet.",
     " Life goes on for now - {PRONOUN/m_c/subject}'ll move into the nursery, but not this moon.",
     " {PRONOUN/m_c/subject/CAP}{VERB/m_c/'re/'s} still full of energy and moving to the nursery would feel stifling - maybe next moon.",
-    " Surely it's not time to move into the nursery, though."
+    " Surely it's not time to move into the nursery, though.",
+	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet though and insists {PRONOUN/m_c/subject} can keep working.",
```

```suggestion
	  " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet and insists {PRONOUN/m_c/subject} can keep working.",
```
"insists" needs a verb tag

---

```suggestion
    " {PRONOUN/m_c/subject/CAP} couldn't imagine being trapped in the nursery yet, and {VERB/m_c/insist/insists} {PRONOUN/m_c/subject} can keep working.",
```